UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/Controllers/RadioNavigationalWarningsAdminController.cs

```diff
@@ -7,6 +8,7 @@
 
 namespace UKHO.MaritimeSafetyInformationAdmin.Web.Controllers
 {
+    [Authorize(Roles = "rnw-admin")]
```

Comment:
I can't see any tests anywhere to confirm that an unauthorised user cannot call these admin end-points.  Where are they please?

Reply:
"WhenICallGetRadioNavigationWarningsForAdmin_ThenCheckIfUserHasCorrectRole" test case added